ios/RCTWebRTC/WebRTCModule+RTCMediaStream.m

```diff
@@ -198,6 +239,15 @@ - (RTCVideoTrack *)createVideoTrack:(NSDictionary *)constraints {
 {
   RTCMediaStream *stream = self.localStreams[streamID];
   if (stream) {
+    for (RTCVideoTrack *track in stream.videoTracks) {
```

Comment:
I think you got a weird merge, this code was refactored.

Reply:
reverted